src/css/profile/mobile/base.less

```diff
@@ -80,6 +80,7 @@
 @content-area-line-color: #d6d6d6;
 @dropdown-list-box-shadow-color: fade(@_black, 35%);
 @calendar-weekend-day-color: #c95151;
+@calendar-select-primary-color: #0381fe;
```

Comment:
I think you don't need this variable. You can use  just `--primary-color`

Reply:
done.